awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -495,6 +577,42 @@ function JobTemplateForm({
                       setAllowCallbacks(checked);
                     }}
                   />
+                  <Checkbox
+                    aria-label={i18n._(t`Enable Webhook`)}
+                    label={
+                      <span>
+                        {i18n._(t`Enable Webhook`)}
+                        &nbsp;
+                        <FieldTooltip
+                          content={i18n._(
+                            t`Enable webhook for this workflow job template.`
+                          )}
+                        />
+                      </span>
+                    }
+                    id="wfjt-enabled-webhooks"
+                    isChecked={
+                      Boolean(webhookServiceField.value) || enableWebhooks
+                    }
+                    onChange={checked => {
+                      setEnableWebhooks(checked);
```

Comment:
this function feels a bit verbose, could we maybe make it a bit less by doing something like

`webhookServiceHelpers.setValue(!checked ? '' : webhookServiceMeta.initialValue);`

etc.

Reply:
yeah I can do that